src/QMCDrivers/WFOpt/QMCFixedSampleLinearOptimizeBatched.h

```diff
@@ -37,8 +37,20 @@ namespace qmcplusplus
  * generated from VMC.
  */
 
-class QMCFixedSampleLinearOptimizeBatched : public QMCLinearOptimizeBatched,
-                                            private NRCOptimization<QMCTraits::RealType>
+class QMCFixedSampleLinearOptimizeBatched;
+
+// Wrapper class for evaluation of Func so QMCFixedSampleLinearOptimizeBatched
+// can avoid inheriting from NRCOptimization
+class OptimizationFunction : public NRCOptimization<QMCTraits::RealType>
```

Comment:
Better to put it on a separate file. `OptimizationFunction` based on the name. I got confused with costfunction. Any way to make it less confusing?